src/fhost_fpga.py

```diff
@@ -371,8 +407,9 @@ def write_delay(self, source_name, delay=0, delta_delay=0, phase_offset=0,
 
         if delta_phase_offset != 0:
             if act_delta_phase_offset == 0:
-                LOGGER.info('Requested phase offset delta is too slow for this configuration. '
-                    'Setting phase offset change to zero.')
+                LOGGER.info('Requested phase offset delta is too slow for this'
+                            ' configuration.'
+                                'Setting phase offset change to zero.')
```

Comment:
I agree, there needs to be a standard. If you're willing to swalllow it wholesale, CAM wrote up a code style guide that you can use for free. On line lengths we settled on 90.

In any case, making style changes in the absence of an agreed style guide tends not to be very productive in the longer term.